jax_lensing/models/convdae.py

```diff
@@ -365,7 +365,8 @@ def __init__(self,
                      channels_per_group=(32, 64, 128, 128),
                      use_projection=(True, True, True, True),
                      # 320 -> 160 -> 80 -> 40
-                     strides=(2, 2, 2, 2),
+                     # 360 -> 180 -> 90 -> 45
+                     strides=(2, 2, 2, 1),
```

Comment:
maybe we shouldn't modify this line if for now  we are using the convdae2 for the 360x360 case